redback/constraints.py

```
@@ -259,6 +308,145 @@ def piecewise_polytrope_eos_constraints(parameters):
+                gamma_3=inputs[5][array_index])
+            mtov = eos_model.maximum_mass()
+            masses = np.array([inputs[0][array_index], inputs[1][array_index]])
+            tidal_deformability, _ = eos_model.lambda_of_mass(central_pressure=central_pressure, mass=masses)
+            ejecta_relation = ejr.TwoComponentBNS(
+                mass_1=inputs[0][array_index], mass_2=inputs[1][array_index],
+                lambda_1=tidal_deformability[0], lambda_2=tidal_deformability[1],
+                mtov=mtov, zeta=inputs[6][array_index])
+            dynamical_mej[index] = ejecta_relation.dynamical_mej
+            disk_wind_mej[index] = ejecta_relation.disk_wind_mej
+            ejecta_velocity[index] = ejecta_relation.ejecta_velocity
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)